### DISCUSSION TITLE
Portable version for Floorp?

### DISCUSSION THREAD

**Discussion by akash07k** [Floorp-Projects/Floorp#724] [Feature Request]
Hi all,
I request that there should be a portable version of Floorp so that we can use it without installing as well.
Any ideas on that?

**Comment by creeper-0910** [comment]
https://floorp.app/en/download/

**Comment by akash07k** [comment]
Thanks, got it.

**Reply by creeper-0910** [reply]
Portable version of 11.8.1 is in build. It will be completed within a few minutes.

**Comment by minus21grams** [comment]
I’ve noticed it takes a day or so before the [portable repository](https://github.com/Floorp-Projects/Floorp-Portable/releases) is updated...

**Comment by creeper-0910** [comment]
That is because it is being updated manually. Sorry.

**Reply by minus21grams** [reply]
No need to apologize, I only mentioned it to spare everyone (users, maintainers, moderators) from similar questions/requests in the future.
I’m a huge fan of portable apps (I wish all developers offered portable versions of their software - Do you hear that [Brave](https://github.com/brave/brave-browser)?), and was waiting patiently knowing it would take a few more hours.
